28. Implement strStr().py

```diff
@@ -0,0 +1,5 @@
+class Solution:
+    def strStr(self, haystack, needle):
+        if needle in haystack:
+            return haystack.index(needle)

```

Comment:
This is not a solution, the same as buying a chair is not the same as making the chair.